### DISCUSSION TITLE
Como puedo poner minúsculas en el code de una property cuando la creo

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#422] [2. Questions and Answers]
Hola a todos

El tema es el siguiente:

Cuando uso el PBCreateProperty, el campo display y code ocupo el mismo, el problema es si mi campo de "texto" coloco con mayúsculas el display no tendría problemas, pero en el code me quedaría con mayúsculas y podría generar algún impacto en el futuro.

Ejemplo: si tengo el creador de "frutas" y creo "Naranja", el display y code serán los mismos, por lo que me gustaría saber cómo dejar Display: Naranja y Code: naranja

Espero que me ayuden.

Gracias =>

**Comment by agallardol** [comment]
@NicolasPLG, puedes utilizar un step con código custom de javascript y utilizar el método toLowerCase para intentar dejar tus codes en el formato correcto

**Reply by NicolasPLG** [reply]
Se puede usar toLowerCase(JOIN##($contexto de la propiedad) en el code del bot crear property?

y si es por CCJS cómo sería el formato del toLowerCase?

Atento =>

**Reply by NicolasPLG** [reply]
Pensé en colocar en el code de la rutina esto, pero no me funcionó
$VALUE#data|[find=>identifier=ingrese_property]|process|[toLowerCase=>*]
Si es por CCJS, debería usar un update property o un patch o eso estará dentro del ccjs
